wandb/fastai/__init__.py

```diff
@@ -29,11 +29,13 @@
         learn.fit(..., callbacks=WandBCallback(learn, ...))
 '''
 import wandb
-import matplotlib
-matplotlib.use('Agg')  # non-interactive back-end (avoid issues with tkinter)
-import matplotlib.pyplot as plt
 from fastai.callbacks import TrackerCallback
 from pathlib import Path
+import random
+try:
+    import matplotlib.pyplot as plt
```

Comment:
Do you think we should set the backend to Agg here?  I've definitely run into issues especially in docker containers where matplotlib just bombs out because we're headless.  I assume if someone wants to use a tk backend they can override our call here...

Reply:
Good point, I had it before and removed the dependency of matplotlib at some point until I needed it again. I forgot to put back agg backend